sftpretty/__init__.py

```diff
@@ -648,7 +648,7 @@ def put_d(self, localdir, remotedir, callback=None, confirm=True,
         :raises IOError: if remotedir doesn't exist
         :raises OSError: if localdir doesn't exist
         '''
-        self.mkdir_p(remotedir)
+        self.mkdir_p(Path(remotedir).joinpath(localdir.stem).as_posix())
```

Comment:
Sorry just had a chance to check this again. So on this line localdir is still a string.

Reply:
@byteskeptical 👋